dipy/viz/horizon/util.py

```diff
@@ -0,0 +1,10 @@
+def check_img_shapes(images):
+    """
+    Checks if the images have same shapes
+    """
+    base_shape = images[0][0].shape[:3]
+    for img in images:
+        data, _ = img
+        if base_shape != data.shape[:3]:
+            return False
+    return True
```

Comment:
This code means that we activate `synchronize_slices` when you have 1 image when it is not needed. 

you should check if you have at least 2 images.